### DISCUSSION TITLE
Object Tracking

### DISCUSSION THREAD

**Discussion by JDface** [roboflow/supervision#507] [Built with supervision]
I want to know if tracking is done by following an object wherever it goes or if it only tracks within the vicinity of the camera. let's say there are 10 cameras and the object is walking from camera 1 and as it passes to camera 2, 3, 4, and so on, does the tracking follow all the way from camera 1 to camera 10?

Does your supervision provide tagging the particular object that you want to follow from start to end?

**Comment by maddust** [comment]
> I want to know if tracking is done by following an object wherever it goes or if it only tracks within the vicinity of the camera. let's say there are 10 cameras and the object is walking from camera 1 and as it passes to camera 2, 3, 4, and so on, does the tracking follow all the way from camera 1 to camera 10?
> 
> Does your supervision provide tagging the particular object that you want to follow from start to end?

if I'm not mistaken, unfortunately, that is not currently possible using only object detection tracking algorithms, object tracking is based on the spatial relation of an object by scene and cannot relate the same object in other video scenes.

however, what you are trying to do can possibly be achieved by cropping the object on each camera that has been seen, and then using an image similarity method and model you can probably identify the same object in multiple cameras or at least narrow down your search.

**Reply by JDface** [reply]
Thanks for your reply. So it is like post analysis, where I have to stitch video to track that object?

**Reply by netrosec** [reply]
you don't have to stitch it. you just used a re-id model (which crops the detection and generates an embedding for it before using a knn to reidentify it in other cameras. 
so yes its is a post-process after the initial detection.
